src/status_im/chat/events/shortcuts.cljs

```diff
@@ -16,6 +16,9 @@
   (merge {:db (-> db
                   (send.events/set-and-validate-amount-db (:amount params) :ETH 18)
                   (choose-recipient.events/fill-request-details (transaction-details contact))
+                  (assoc-in [:wallet :send-transaction :id] nil)
```

Comment:
`(update-in [:wallet :send-transaction] dissoc :id :password :wrong-password?)` or we need to have `nil` there?

Reply:
No idea really, I just saw `nil`s in the repl in the "happy" use case and used that without thinking too much.
It really shouldn't make a difference, so I'll give it a shot since it's a lot cleaner